draft/spec/index.html

```diff
@@ -1296,172 +1298,162 @@ <h2>Moab in an OCFL Object</h2>
-        │       └── manifests
-        │           ├── fileInventoryDifference.xml
-        │           ├── manifestInventory.xml
-        │           ├── signatureCatalog.xml
-        │           ├── versionAdditions.xml
-        │           └── versionInventory.xml
+        │     ├── data
+        │     │   └── metadata
+        │     │       ├── contentMetadata.xml
+        │     │       ├── embargoMetadata.xml
+        │     │       ├── events.xml
+        │     │       ├── identityMetadata.xml
+        │     │       ├── provenanceMetadata.xml
+        │     │       ├── relationshipMetadata.xml
+        │     │       ├── rightsMetadata.xml
+        │     │       ├── versionMetadata.xml
+        │     │       └── workflows.xml
+        │     └── manifests
+        │         ├── fileInventoryDifference.xml
+        │         ├── manifestInventory.xml
+        │         ├── signatureCatalog.xml
+        │         ├── versionAdditions.xml
+        │         └── versionInventory.xml
         └── v0003
-            └──content
-                ├── data
-                │   └── metadata
-                │       ├── contentMetadata.xml
-                │       ├── descMetadata.xml
-                │       ├── embargoMetadata.xml
-                │       ├── events.xml
-                │       ├── identityMetadata.xml
-                │       ├── provenanceMetadata.xml
-                │       ├── rightsMetadata.xml
-                │       ├── technicalMetadata.xml
-                │       ├── versionMetadata.xml
-                │       └── workflows.xml
-                └── manifests
-                    ├── fileInventoryDifference.xml
-                    ├── manifestInventory.xml
-                    ├── signatureCatalog.xml
-                    ├── versionAdditions.xml
-                    └── versionInventory.xml
+              ├── data
+              │   └── metadata
+              │       ├── contentMetadata.xml
+              │       ├── descMetadata.xml
+              │       ├── embargoMetadata.xml
+              │       ├── events.xml
+              │       ├── identityMetadata.xml
+              │       ├── provenanceMetadata.xml
+              │       ├── rightsMetadata.xml
+              │       ├── technicalMetadata.xml
+              │       ├── versionMetadata.xml
+              │       └── workflows.xml
+              └── manifests
+                  ├── fileInventoryDifference.xml
+                  ├── manifestInventory.xml
+                  ├── signatureCatalog.xml
+                  ├── versionAdditions.xml
+                  └── versionInventory.xml
 </pre>
         <p>
-            An OCFL inventory that tracks the /data directory would include a manifest comprised as follows:
+            An OCFL inventory that tracks the /data directory would include a manifest comprised as follows.
+            Note the absence of the /manifests directory, as we are not encapsulating the Moab object in an OCFL object,
```

Comment:
`/manifests` --> `<code>manifests</code>`

Reply:
Resolved in new commit